apps/sms/views/inbox/js/startup.js

```diff
@@ -47,6 +47,10 @@
         initLazyDependencies();
       });
 
+      InboxView.once('fully-loaded', () => {
+        App.setReady();
```

Comment:
I think it's not related to the issue, right? It seems just for fixing split mode case.
BTW do you think we'll need unit test for it?